packages/gatsby-source-lever/src/gatsby-node.js

```diff
@@ -3,6 +3,37 @@ const normalize = require(`./normalize`)
 
 const typePrefix = `lever__`
 
+exports.createSchemaCustomization = async ({ actions }) => {
+  const typeDefs = `
+    type lever implements Node @dontInfer {
+      additional: String
+      additionalPlain: String
+      applyUrl: String
+      createdAt: Date @dateformat
+      description: String
+      descriptionPlain: String
+      hostedUrl: String
+      lever_id: String
+      text: String
+      categories: LeverCategories
+      lists: [LeverList]
```

Comment:
We should lowercase `LeverCategories` and `LeverList` here as well

Reply:
done